rules.d/60-persistent-storage.rules

```diff
@@ -93,6 +93,9 @@ ENV{DEVTYPE}=="disk", DEVPATH!="*/virtual/*", IMPORT{builtin}="path_id"
 KERNEL=="mmcblk[0-9]boot[0-9]", ENV{DEVTYPE}=="disk", ENV{ID_PATH}=="?*", SYMLINK+="disk/by-path/$env{ID_PATH}-boot%n"
 KERNEL!="mmcblk[0-9]boot[0-9]", ENV{DEVTYPE}=="disk", ENV{ID_PATH}=="?*", SYMLINK+="disk/by-path/$env{ID_PATH}"
 ENV{DEVTYPE}=="partition", ENV{ID_PATH}=="?*", SYMLINK+="disk/by-path/$env{ID_PATH}-part%n"
+# ATA device compatible links
+KERNEL!="mmcblk[0-9]boot[0-9]", ENV{DEVTYPE}=="disk", ENV{ID_PATH}=="*ata*", PROGRAM="/bin/sh -c 'compath=$env{ID_PATH};echo ${compath%\.[0-9]*}'",ENV{ID_PATH_ATA_COMPAT}="%c", SYMLINK+="disk/by-path/$env{ID_PATH_ATA_COMPAT}"
+ENV{DEVTYPE}=="partition", ENV{ID_PATH}=="*ata*", PROGRAM="/bin/sh -c 'compath=$env{ID_PATH};echo ${compath%\.[0-9]*}'", ENV{ID_PATH_ATA_COMPAT}="%c", SYMLINK+="disk/by-path/$env{ID_PATH_ATA_COMPAT}-part%n"
```

Comment:
please, no. No shelling out here. if this is desirable, implement generating the env var in path_id.c